wsmaster/codenvy-hosted-api-organization/src/main/java/com/codenvy/organization/shared/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.organization.shared;
+
+/**
+ * Constants for Organization API
+ *
+ * @author Sergii Leschenko
+ */
+public final class Constants {
+    public static final String LINK_REL_SELF                    = "self";
+    public static final String LINK_REL_GET_CHILD_ORGANIZATIONS = "get child organizations";
```

Comment:
This is not actually the right way to define rels, [see](https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/Constants.java#L22-L34)